syntax = "proto3";

package raw_data;

message RawCarryRequest {
    string symbol = 1;
}

message RawCarryResponse {
    bytes series = 1;
}

service RawCarry {
    rpc get_raw_carry (RawCarryRequest) returns (RawCarryResponse);
}